apps/landingPage/src/components/Info/Features.js

```diff
@@ -0,0 +1,72 @@
+// @flow
+
+import React from 'react';
+import { Heading } from '@kiwicom/orbit-components/lib/';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+import styled from 'styled-components';
+
+import { BREAKPOINTS } from '../../mediaQueriesConfig';
+
+const features = ['Search', 'Booking', 'MMB', 'Payments'];
+
+const renderCards = (features: $ReadOnlyArray<string>) => {
+  return features.map(feature => (
+    <CardWrapper key={feature}>
+      <Heading type="title2">{feature}</Heading>
+    </CardWrapper>
+  ));
+};
+
+export default function Features() {
+  return (
+    <Container>
+      <Title>
+        <Heading>Features</Heading>
+      </Title>
+      <CardListWrapper>{renderCards(features)}</CardListWrapper>
```

Comment:
Also, I don't understand why renderCards is a function. It looks and acts like a component. 
What about 
```
const RenderCards = ({features}: Props) => {
return features.map(feature => (
    <CardWrapper key={feature}>
      <Heading type="title2">{feature}</Heading>
    </CardWrapper>
  ));
}

...
 <CardListWrapper><RenderCards features={features} /></CardListWrapper>
```

Reply:
Ok! changed to the component